app/models/edition.rb

```diff
@@ -463,11 +463,15 @@ def latest_link_check_report
   end
 
   def remove_line_separator_character
-    return unless respond_to?(:parts)
+    return unless respond_to?(:parts) || respond_to?(:body)
```

Comment:
You can probably remove this line, as the `if` and `elsif` with have the same effect of returning `nil` if neither method are available.

Reply:
Done 👍